src/ucs/arch/generic/cpu.c

```diff
@@ -11,24 +11,28 @@
 #include <ucs/arch/cpu.h>
 #include <ucs/sys/sys.h>
 #include <ucs/sys/string.h>
+#include <ucs/type/init_once.h>
 
 #define UCS_CPU_CACHE_FILE_FMT     "/sys/devices/system/cpu/cpu%d/cache/index%d/%s"
 #define UCS_CPU_CACHE_LEVEL        "level"
 #define UCS_CPU_CACHE_TYPE         "type"
 #define UCS_CPU_CACHE_SIZE         "size"
 
+#define UCS_CACHE_ENTRY_INIITIALIZER(_level, _type) \
```

Comment:
after reducing # of fields, probably this macro can go away

Reply:
ok, removed